docs/red_eyes_mainnet_tutorial.rst

```diff
@@ -0,0 +1,165 @@
+
+Raiden has a Restful API with URL endpoints corresponding to actions that users can perform with their channels. The endpoints accept and return JSON encoded objects. The API URL path always contains the API version in order to differentiate queries to different API versions. All queries start with: ``/api/<version>/`` where ``<version>`` is an integer representing the current API version.
+
+Before getting started with this tutorial, please see the :doc:`Installation Guide <overview_and_guide>`, to make sure that Raiden is correctly installed and running.
+
+.. _join-token-network:
+
+Joining a token network
+==============================
+The first thing we need to do is to join a token network. In this case we want to join the Raiden token (`RDN <https://etherscan.io/token/0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6>`_) network.
+
+*Note*: 1 RDN == 2**18 Rei. For the sake of readability and simplicity all token values in this tutorial are denominated in Rei and not RDN.
+
+In order to do so, we need the address of the token and the initial amount of tokens that we want to join the network with:
+
+.. http:example:: curl wget httpie python-requests
+
+    PUT /api/1/connections/0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6 HTTP/1.1
+    Host: localhost:5001
+    Content-Type: application/json
+
+    {
+        "funds": 20000
+    }
+
+This automatically connects our node to 3 other nodes in the network with 20000 / 5 Rei tokens in each channel.
+
+We're now ready to start sending RDN tokens using the Raiden Network.
+
+In case we know of a specific address in the network that we will do frequent payments to, we can open a channel directly to this address by doing the following:
+
+.. http:example:: curl wget httpie python-requests
+
+   PUT /api/1/channels HTTP/1.1
+   Host: localhost:5001
+   Content-Type: application/json
+
+   {
+       "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+       "token_address": "0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6",
+       "balance": 2000,
+       "settle_timeout": 600
+   }
+
+At this point the specific value of the ``balance`` field isn't too important, since it's always possible to :ref:`deposit more tokens <topping-up-a-channel>` to a channel if need be.
+
+Successfully opening a channel returns the following information:
+
+.. sourcecode:: http
+
+   HTTP/1.1 201 CREATED
+   Content-Type: application/json
+
+   {
+       "channel_identifier": "0xfb43f382bbdbf209f854e14b74d183970e26ad5c1fd1b74a20f8f6bb653c1617",
+       "token_network_identifier": "0x3C158a20b47d9613DDb9409099Be186fC272421a",
+       "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+       "token_address": "0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6",
+       "balance": 2000,
+       "state": "opened",
+       "settle_timeout": 600,
+       "reveal_timeout": 10
+   }
+TODO: update token_network_identifier once it's known
+
```

Comment:
Should we also mention that one can query the open channels with `GET /channels/<token>`?